app/controllers/UploadWrittenAuthorisationController.scala

```diff
@@ -95,12 +95,37 @@ class UploadWrittenAuthorisationController @Inject()(
     }
 
   private val maxFileSize = 10485760
+  private val allowedTypes = Set(
```

Comment:
We need to add the same validation in the operational UI I believe....
Is it worth doing this maybe in `binding-tariff-filestore`, so that is used by both front-end microservices?

Reply:
We need in both, but this does not make filestore the place where to validate. We could make a third service which could allow any type of file for example.